app/controllers/application_controller.rb

```diff
@@ -19,6 +20,25 @@ def after_sign_out_path_for(_resource)
 
   protected
 
+  def check_requirements
```

Comment:
Instead of `fix_secrets = false` and then the if, do:
```ruby
fix_secrets = true if Rails.application.secrets.secret_key_base == "CHANGE_ME"
```
The same for `fix_ssl`. If the variable is not defined it will be nil, therefore evaluating to false anyways ;) Moreover, rubocop is complaining about the final if statement. Instead of doing that, you should:
```ruby
return unless fix_secrets || fix_ssl
```
And then proceed with the code.

Moreover, I'd appreciate if you could add documentation to this method :)